modules/docs/mdx/9.0-UPGRADE-GUIDE.mdx

```diff
@@ -92,6 +93,25 @@ using the `Flex` component and to replace the `spacing` prop with the `gap` prop
 > **Note**: If you are consuming `StackProps` for a custom component, please make sure to change the
 > `spacing` prop to the `gap` prop
 
+---
+
+### Menu in Preview
+
+We've removed the `Menu` component that was in `@workday/canvas-kit-preview-react` and now will be
+replaced with the `Menu` component in `@workday/canvas-kit-react`. Please see the `Menu`
+documentation
+[here](https://workday.github.io/canvas-kit/?path=/docs/components-popups-menu--basic).
+
+For the migration guide from `Menu` in `@workday/canvas-kit-preview-react` to the `Menu` component
+in `@workday/canvas-kit-react`, see [this](https://github.com/Workday/canvas-kit/discussions/2063)
+discussion.
+
+If you would like to see the differences between `DeprecatedMenu` and the `Menu` compound component,
+see [this](https://codesandbox.io/s/deprecatedmenu-to-menu-migration-gwi1ov) codesandbox example.
+
+> **Note**: There will be no codemod to replace this component due to varying differences in API and
+> `Menu` being a compound component.
```

Comment:
It was easier to have one big suggestion instead of several smaller ones. A few notes:

- To @mannycarrera4's point [here](https://github.com/Workday/canvas-kit/pull/2070#discussion_r1130071586) and [here](https://github.com/Workday/canvas-kit/pull/2070#discussion_r1130073797): we typically refer to the packages as "Preview" and "Main" (without backticks) for brevity. Sometimes, we'll write "the Preview package" or "the Main package." It is nice being explicit about the package names, though -- maybe we could add a short section at the top of the upgrade guide which defines Main, Preview, and Labs.
- When linking to other docs, you'll want to use the routes defined in [routes.js](https://github.com/Workday/canvas-kit/blob/master/.storybook/routes.js). For example, to link to Menu, use `/components/popups/menu/` instead of `https://workday.github.io/canvas-kit/?path=/docs/components-popups-menu--basic`. `/components/popups/menu/` will work correctly on the Canvas Site, whereas the `workday.github.io` link will take someone on the site to the Menu docs in Storybook instead of the Menu docs on the site.
- No need to include the codesandbox link here since it's already included in the linked migration discussion. **However**, I would consider moving the contents of that discussion into the upgrade guide itself. I took a quick look at the discussion, and there's some cleanup to be done, but I think it could go directly into the upgrade guide (at which point the codesandbox link would be added back to the guide). What do you think?

```suggestion
### Menu (Preview)

We've removed the `Menu` component in Preview. We recommend using the
[Menu](/components/popups/menu/) in the Main package instead; it is a
[compound component](/getting-started/for-developers/resources/compound-components/) and offers a
more flexible API. Please refer to this
[discussion](https://github.com/Workday/canvas-kit/discussions/2063) for instructions on how to
migrate from the `Menu` in Preview to the `Menu` in Main.

This change is **not** handled by the codemod due to the differences in API between the `Menu` in
Preview and the `Menu` in Main.
```